spec/support/factory_bot.rb

```diff
@@ -1,11 +1,4 @@
-def upload_file(file)
-  content = File.read(file)
-  tempfile = Tempfile.open
-  tempfile.write content
-  tempfile.rewind
-
-  Rack::Test::UploadedFile.new(tempfile)
-end
```

Comment:
This is a weird place for this to be defined! Global helper, in an unrelated file.

Rather than put in a helper file, I replaced its usage with [`fixture_file_upload`](https://apidock.com/rails/v7.1.3.4/ActionDispatch/TestProcess/FixtureFile/fixture_file_upload). 

By default, the method looks in `spec/fixtures/files`, so that is why I moved all of the .csv fixtures into that directory. Moving them was easier than figuring out how to config the fixture files directory, and felt like the standard way to do it.